src/System.Private.Xml/tests/XmlSchema/XmlSchemaSet/TC_SchemaSet_AnyAttribute.cs

```diff
@@ -240,6 +240,30 @@ public void v3(string ns, string attrNs, string expectedNs)
             CompareWildcardNamespaces(expectedNs, attributeWildcard.Namespace);
         }
 
+        [Fact]
+        //[Variation(Desc = "new instance return null namespace (issue #41704")]
+        public void v4()
```

Comment:
The existing test names are from probably several test suites ago.  The preferred format now is to have the name convey the purpose of the test.  Then you can delete the commented out Variation attribute :smile:.